.travis.yml

```diff
@@ -1,48 +1,81 @@
 # Travis Continuous Integration Configuration File For ROS Control Projects
-# Author: Dave Coleman
-language:
-  - cpp
-  - python
-python:
-  - "2.7"
+# Author: Dave Coleman, Enrique Fernandez
+sudo: required
+dist: xenial
```

Comment:
Thanks for the note! Probably we are not getting (I definitely don't!) how Travis works haha. 
I've found this:
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

Perhaps we need to change the scripts to use docker on Travis and get a 1604 environment that way? If that's possible at all...